### ISSUE TITLE
openssl compilation error

### ISSUE THREAD

**Issue by 3wille** [pablocouto/crest#2]
Hi there,
I run into an error when trying to compile rust's ``openssl-sys v0.7.17``. The openssl crates are required by ``hyper`` and using a new version of ``hyper`` and with it ``openssl`` resolves the issues. You might want to update to a newer ``hyper`` version.

stderr output is 
```
thread 'main' panicked at 'explicit panic', /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/gcc-0.3.46/src/lib.rs:1200
```
If you need any further information, feel free to ask and thanks for your time :)

**Comment by pablocouto** [comment]
Hi @3wille, and thank you for opening this issue.

I couldn’t reproduce the error from `openssl-sys`, but got failures from other dependencies. I’ll try to update deps in general, and see if that brings the crate to a usable state.

**Comment by 3wille** [comment]
Hi,
yeah just hit me, if you're done with that or have something for me to test.

**Comment by pablocouto** [comment]
As of commit 791d727175793c880c844012020f65bc1783a00f (published version 0.3.4), the crate should compile without errors on Rust 1.18.0. @3wille, if you don’t come against this issue anymore, please confirm it so I can close the ticket.

**Comment by pablocouto** [comment]
Closing due to inactivity. Feel free to reopen the issue if necessary.
